topsort-api-v2.yml

```diff
@@ -466,6 +508,206 @@ components:
           description: >
             Human-readable explanation of or details about the error.
             The string for a given error may change over time; code should not parse or dispatch based on particular values for this field.
+    Placement:
+      type: object
+      required:
+        - path
+      properties:
+        path:
```

Comment:
Could we request that this is not empty? Some marketplaces are setting it to the empty string.

Reply:
Good catch.